### ISSUE TITLE
[R6] Entities timings problem

### ISSUE THREAD

**Issue by lucas2107** [VolmitSoftware/React#142]
Same error since one month with R6

https://hastebin.com/oyoxizamep.css

**Comment by cyberpwnn** [comment]
```
[17:55:11] [Surge World Monitor/ERROR]: stopTiming called async for world - chunkLoad - Entities
java.lang.Throwable
 	at co.aikar.timings.TimingHandler.stopTiming(TimingHandler.java:114)
 	at net.minecraft.server.v1_8_R3.ChunkRegionLoader.loadEntities(ChunkRegionLoader.java:424)
 	at org.bukkit.craftbukkit.v1_8_R3.chunkio.ChunkIOProvider.callStage2(ChunkIOProvider.java:46)
 	at org.bukkit.craftbukkit.v1_8_R3.chunkio.ChunkIOProvider.callStage2(ChunkIOProvider.java:1)
 	at org.bukkit.craftbukkit.v1_8_R3.util.AsynchronousExecutor.skipQueue(AsynchronousExecutor.java:337)
 	at org.bukkit.craftbukkit.v1_8_R3.util.AsynchronousExecutor.getSkipQueue(AsynchronousExecutor.java:295)
 	at org.bukkit.craftbukkit.v1_8_R3.chunkio.ChunkIOExecutor.syncChunkLoad(ChunkIOExecutor.java:16)
 	at net.minecraft.server.v1_8_R3.ChunkProviderServer.getChunkAt(ChunkProviderServer.java:135)
 	at net.minecraft.server.v1_8_R3.ChunkProviderServer.getChunkAt(ChunkProviderServer.java:117)
 	at org.bukkit.craftbukkit.v1_8_R3.CraftWorld.getChunkAt(CraftWorld.java:145)
 	at org.bukkit.craftbukkit.v1_8_R3.CraftWorld.getBlockAt(CraftWorld.java:91)
 	at org.bukkit.craftbukkit.v1_8_R3.CraftChunk.getTileEntities(CraftChunk.java:114)
 	at react.surge.server.WorldMonitor.sampleTileCount(WorldMonitor.java:222)
 	at react.surge.server.WorldMonitor.sample(WorldMonitor.java:156)
 	at react.surge.server.WorldMonitor.run(WorldMonitor.java:52)
[17:55:11] [Surge World Monitor/ERROR]: stopTiming called async for world - chunkLoad - TileEntities
```

**Comment by cyberpwnn** [comment]
Fixed in 6.27159a
